server/services/navigationService.ts

```diff
@@ -13,14 +13,24 @@ type NavigationConfig = {
 
 export default class NavigationService {
```

Comment:
Can we add an unit test case to navigationService.test.ts for this.

Reply:
Added unit tests